aten/src/ATen/core/jit_type.h

```diff
@@ -1600,6 +1600,39 @@ CAFFE2_API TypePtr tryEvalTypeVariables(TypePtr type, TypeEnv& type_env);
 
 CAFFE2_API bool elementTypeCanBeInferredFromMembers(const TypePtr& elem_type);
 
+enum class AttributeKind {
```

Comment:
nit: this enum and class are relatively self-explantory, but a small comment may be useful for the future